frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx

```diff
@@ -188,13 +188,21 @@ export const OperatorHubList: React.FC<OperatorHubListProps> = ({
             (key) => InfraFeatures[key],
           );
 
+          let tokenAuthSupport = 'false';
+          if (tokenAuthAWS === 'true' && isAWSSTSCluster(cloudCredential, infra, auth)) {
+            tokenAuthSupport = 'true';
+          } else if (tokenAuthAzure === 'true' && isAzureWIFCluster(cloudCredential, infra, auth)) {
+            tokenAuthSupport = 'true';
+          }
+
           const featuresAnnotationsObjects = [
             { key: InfraFeatures.Disconnected, value: disconnected },
             { key: InfraFeatures.FipsMode, value: fipsCompliant },
             { key: InfraFeatures.Proxy, value: proxyAware },
             { key: InfraFeatures.cnf, value: cnf },
             { key: InfraFeatures.cni, value: cni },
             { key: InfraFeatures.csi, value: csi },
+            { key: InfraFeatures.TokenAuth, value: tokenAuthSupport },
```

Comment:
My memory on this is fuzzy as it has been awhile and this has changed a bit.  `featuresAnnotationsObjects` really only exists in order to override legacy labels.  Since `tokenAuth*`s are new, I would not include them here.  I would preserve the existing functionality where you push the value on to `infrastructureFeatures`.